sources/models/shared-defaults/oci-capabilities.toml

```diff
@@ -0,0 +1,17 @@
+[settings.oci-defaults.capabilities]
+# These values were sourced from the output of `ctr oci spec`, then
+# modified for Bottlerocket.
+audit-write = true
+chown = true
+dac-override = true
+fowner = true
+fsetid = true
+kill = true
+mknod = true
+net-bind-service = true
+net-raw = false
+setgid = true
+setfcap = true
+setpcap = true
+setuid = true
+sys-chroot = true
```

Comment:
How we acquired the default OCI spec values:

We ran `ctr oci spec` on a Bottlerocket node, which printed the default 1.0.2-dev values (latest as per https://github.com/opencontainers/runtime-spec/blob/main/specs-go/version.go )

Reply:
From there, we removed CAP_NET_RAW as per the following discussion: https://github.com/bottlerocket-os/bottlerocket/issues/1703#issuecomment-1240834183

Reply:
...but, we had to add CAP_NET_RAW back in because the AWS CNI requires it and previous versions of it do not specify the requirement in the podspec.

I am going to resolve this comment so that it's not a distraction.